com.unity.multiplayer.mlapi/Runtime/NetworkVariable/Collections/NetworkDictionary.cs

```diff
@@ -395,7 +396,7 @@ public bool IsDirty()
                 return false;
             }
 
-            if (m_NetworkBehaviour.NetworkManager.NetworkTime - LastSyncedTime >= (1f / Settings.SendTickrate))
+            if (m_NetworkBehaviour.NetworkManager.LocalTime.FixedTime - LastSyncedTime.FixedTime >= (1f / Settings.SendTickrate))
```

Comment:
We're comparing a double and a float. :-) Probably not a huge deal, but somewhat annoying. Can we use `1.0` or `1d` to do the division in higher precision and compare double to double ?

Reply:
Good catch, yeah this should be 1.0 and a left over oversight, will fix